.eslintrc.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
   rules: {
     'jsx-a11y/href-no-hash': 'off',
     'no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-expressions': 'off',
```

Comment:
It looks like this rule is turned on in one of the updated configs, and there are a good number of warnings. Turning off since it's not a style this codebase enforces.